community/contribution.md

```diff
@@ -15,28 +15,34 @@ _This document explains how Axelerant defines contributions with respect to KPIs
 
 An open source community contribution is defined as any piece of documentation (code or otherwise) or any offline activity that is documented and available in the public space for other community members to use.
 
+### Axelerant’s intent
+
+While we understand the above is an open source contribution **Axelerant’s intent is to encourage more meaningful contributions for the community & for the organisation.**
+
+## Important Notes before you continue reading:
```

Comment:
@hussainweb I've asked Nathan to help me condense the document by next week, I think we'll tweak this further then. Till then this can be re-titled to "## Before you continue:". 
BTW this additional information was moved to the top because I noticed no one was reading these.